### ISSUE TITLE
Expected results of the G223 technique should be changed.

### ISSUE THREAD

**Issue by ShamarYarde** [w3c/wcag#2922]
The Expected Results section of the [G223: Using an author-supplied, highly visible focus indicator](https://www.w3.org/WAI/WCAG22/Techniques/general/G223) technique should be changed to: "All three checks are true."

**Comment by detlevhfischer** [comment]
Thank you for making us aware of the error. I have updated the Technique following your suggestion:

https://github.com/w3c/wcag/pull/2929
